tests/terraform/checks/resource/azure/test_KeyVaultEnablesRBAC.py

```diff
@@ -0,0 +1,42 @@
+import os
+import unittest
+
+from checkov.runner_filter import RunnerFilter
+from checkov.terraform.runner import Runner
+from checkov.terraform.checks.resource.azure.KeyVaultEnablesRBAC import check
+
+
+class TestKeyVaultEnablesRBAC(unittest.TestCase):
+
+    def test(self):
```

Comment:
Looks like this test is currently not catching everything correctly based on the last unit-tests run, please fix it and update us :)

Reply:
Hi, 

Is it possible to get the logs please ?

Reply:
https://github.com/bridgecrewio/checkov/actions/runs/8844488068/job/24445326331?pr=6175